python_modules/dagster/dagster/core/storage/alembic/versions/011_wipe_schedules_table_for_0_10_0_sqlite_2.py

```diff
@@ -21,6 +22,13 @@
 
 
 def upgrade():
+    bind = op.get_context().bind
+
+    inspector = reflection.Inspector.from_engine(bind)
+
+    if "sqlite" not in inspector.dialect.dialect_description:
```

Comment:
I'm not sure why, but this script was just in the sqlite schedule directory twice, and also does the potentially destructive action, hence the dialect guard.